vendor/assets/stylesheets/font-awesome/_core.scss

```diff
@@ -0,0 +1,13 @@
+// Base Class Definition
+// -------------------------
+
+.#{$fa-css-prefix} {
+  display: inline-block;
+  font: normal normal normal #{$fa-font-size-base}/1 FontAwesome; // shortening font declaration
+  font-size: inherit; // can't have font-size inherit on line above, so need to override
+  text-rendering: auto; // optimizelegibility throws things off #1094
```

Comment:
Properties should be ordered display, font, font-size, -moz-osx-font-smoothing, -webkit-font-smoothing, text-rendering, transform